src/Neo/Network/P2P/Payloads/WitnessRule.cs

```diff
@@ -37,6 +37,27 @@ public class WitnessRule : IInteroperable, ISerializable
 
         int ISerializable.Size => sizeof(WitnessRuleAction) + Condition.Size;
 
+        public bool Equals(WitnessRule other)
+        {
```

Comment:
It fail with null

Reply:
what do you mean? exception? `WitnessRule` is not `nullable`.

Reply:
Because we haven't nullable enabled, also is not in object and is checked

Reply:
ok, fixed